src/components/SearchResults.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import React from "react";
+import "../styles/searchResults.css";
+
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return <p>No results</p>;
+  } else {
+    return (
+      <div className="results">
+        {results.map((image) => (
+          <img
+            className="card-image"
+            src={image}
+            alt="spaceImage"
```

Comment:
This alt tag isnt all that helpful. Their purpose is to describe what an image is for people using a screen reader, in this case all the images will have this generic label that doesnt help describe anything really. A good next step for future development would be to see if you can get the alt tag for each image from the API and attach it to its image